packages/cozy-harvest-lib/src/components/EditAccountModal.jsx

```diff
@@ -48,6 +48,16 @@ const DumbEditAccountModal = ({
    * vault modal), we delay a bit the appearing of the edit modal via CSS.
    */
   const shouldShow = useTimeout(500)
+
+  const fromReconnectButton = window.location.toString().endsWith('reconnect')
```

Comment:
In banks, the hash history is not used, you should the router context to get a handle on the current route.